nginx/nginx.conf

```
@@ -75,5 +75,8 @@ http {
 		location / {
 			proxy_pass http://sentry;
 		}
+		location /_static/ {
+			proxy_hide_header Content-Disposition;
```

This location does not inherit `proxy_pass http://sentry` from anywhere, with this change, it should have the line itself.
```suggestion
                        proxy_pass http://sentry;
			proxy_hide_header Content-Disposition;
```

---

Thanks, good catch!